lib/rules/block-no-empty/index.mjs

```diff
@@ -75,6 +75,7 @@ const rule = (primary, secondaryOptions, context) => {
 				message: messages.rejected,
 				node: statement,
 				start: statement.positionBy({ index }),
+				end: statement.rangeBy({}).end,
```

Comment:
This isn't particularly pretty or elegant but it is an exact match of what happens implicitly when only providing `start` and omitting `end`.